relay-config/src/config.rs

```diff
@@ -1884,12 +1887,17 @@ impl Config {
         Duration::from_secs(self.values.cache.project_grace_period.into())
     }
 
-    /// Returns the number of seconds during which batchable queries are collected before sending
-    /// them in a single request.
+    /// Returns the duration in which batchable project config queries are
+    /// collected before sending them in a single request.
```

Comment:
"Number of seconds" is incorrect and could be misleading: the function returns a `Duration` and the unit of the value is in milliseconds.